server/api/routes/ssot.js

```diff
@@ -29,12 +29,14 @@ router.get(
   '/getAllParameters/:robotId',
   variableController.retrieveParametersForRobot
 );
+router.delete('/deleteParameters', variableController.deleteMany);
 
 /// RPA ATTRIBUTES ROUTES ///
 router.post('/updateManyAttributes', rpaAttributesController.updateMany);
 router.get(
   '/getAllAttributes/:robotId',
   rpaAttributesController.retrieveAttributesForRobot
 );
+router.delete('/deleteAttributes', rpaAttributesController.deleteMany);
```

Comment:
Same here, maybe `deleteAttributesForParameter`?

Reply:
Once the new API is integrated we will have to rename the route anyway, thats why I think it is okay for now :)